site/app/templates/forum/displayThreadList.twig

```diff
@@ -3,17 +3,25 @@
         <div class="{{ thread.class }}">
             <div class="flex-row"  data-testid="flex-row">
                 <span class="thread-left-cont">
-                    <span class="thread-list-item" data-testid="thread-list-item">
-                        {% if thread.current_user_posted %}
-                            <i class="fas fa-comments" title="You have contributed" aria-label="You have contributed"></i>
-                        {% endif %}
-                        {{ thread.title }}
-                    </span>
+                    <span class="thread-list-item" data-testid="thread-list-item">{{ thread.title }}</span>
                     <span class="thread-content">{{ thread.content }}</span>
 
-                    {% for category in thread.categories %}
-                        <span class="label_forum" style="background-color: {{ category[1] }}">{{ category[0] }}</span>
-                    {% endfor %}
+                    <span class="thread-info-bar">
+                        {% for category in thread.categories %}
+                            <span class="label_forum" style="background-color: {{ category[1] }}">{{ category[0] }}</span>
+                        {% endfor %}
+
+                        {% if is_full_page is not defined or not is_full_page %}
+                            <span class="post-cont">
+                                <i class="fas fa-comments" style="position: relative; top: 2px; margin-right: 9px;" title="Total posts" aria-label="Total posts"></i>
+                                <span data-testid="thread-post-count"
+                                    id="Thread_postCounter_{{thread.thread_id}}"
```

Comment:
Is this ID being used for anything?  If not, let's remove it to prevent confusion when someone refactors this in the future.  Apply throughout as well.